pydir/wcsfit.cpp

```diff
@@ -102,6 +103,10 @@ PYBIND11_MODULE(wcsfit, m) {
     py::class_<astrometry::SubMap, astrometry::PixelMap>(m, "SubMap")
             .def(py::init<list<astrometry::PixelMap *> const &, std::string, bool>());
 
+    py::class_<astrometry::ASTMap, astrometry::PixelMap>(m, "ASTMap")
+            .def(py::init<ast::Mapping, std::string>(),
```

Comment:
```suggestion
            .def(py::init<ast::Mapping const &, std::string>(),
```
If this isn't marked as a reference or pointer we'll get more type-slicing in the pybind11 layer.